src/extensions/scratch3_vizblocks/index.js

```diff
@@ -28,7 +36,27 @@ class Scratch3VizBlocks {
          */
         this._penSkinId = -1;
 
+
+        // initiate canvas and center of coordinate system
+        this._xCenter = -100;
+        this._yCenter = -100;
+        this._width = 200;
+        this._height = 200;
+        this._interval = 20;
```

Comment:
Can we try this scale instead? It utilizes the space better.
```
        this._xCenter = -160;
        this._yCenter = -100;
        this._width = 360;
        this._height = 260;
        this._interval = 20;
```

Reply:
If we use this scale, then the we need to create new costumes in which numbers are larger than 9, but this scale certainly looks better in terms of space utilization.